checks/class-skip-links.php

```diff
@@ -0,0 +1,152 @@
+<?php
+/**
+ * Check if skip links are present on block themes
+ *
+ * @package Theme Check
+ */
+
+/**
+ * Check if skip links are present on block themes
+ */
+class Skip_Links_Check implements themecheck {
+	/**
+	 * Error messages, warnings and info notices.
+	 *
+	 * @var array $error
+	 */
+	protected $error = array();
+
+	/**
+	 * Returns true if the theme is a block theme.
+	 *
+	 * @var array $is_block_theme
+	 */
+	protected $is_block_theme = false;
+
+	/**
+	 * The WP_Theme instance being checked.
+	 *
+	 * @var WP_Theme $wp_theme
+	 */
+	protected $wp_theme = false;
+
+	function set_context( $data ) {
+		if ( isset( $data['theme'] ) ) {
+			$this->wp_theme = $data['theme'];
+			$theme_dir      = $this->wp_theme->get_stylesheet_directory();
+			// Check if the theme has all the required files.
+			if (
+				file_exists( $theme_dir . '/theme.json' ) ||
+				(
+					file_exists( $theme_dir . '/templates/index.html' ) &&
+					file_exists( $theme_dir . '/block-templates/index.html' )
+				)
+			) {
+				$this->is_block_theme = true;
+			}
+		}
```

Comment:
Could we replace this implementation by the use of the core function `wp_is_block_theme()`. Reference:  https://developer.wordpress.org/reference/functions/wp_is_block_theme/